runner/util_linux.go

```diff
@@ -28,8 +29,15 @@ func (e *Engine) killCmd(cmd *exec.Cmd) (pid int, err error) {
 	return
 }
 
-func (e *Engine) startCmd(cmd string) (*exec.Cmd, io.ReadCloser, io.ReadCloser, error) {
-	c := exec.Command("/bin/sh", "-c", cmd)
+func (e *Engine) startCmd(cmd string) (c *exec.Cmd, err error) {
+	c = exec.Command("/bin/sh", "-c", cmd)
 	f, err := pty.Start(c)
```

Comment:
It appears that `f.Close()` must be called before exit or the process will be stuck.